docs/src/explanations/recipes.md

```diff
@@ -135,6 +135,14 @@ function default_theme(scene, ::MyPlot)
 end
 ```
 
+Furthermore, you can control which kind the axis used for the plot by
+
+```julia
+Makie.args_preferred_axis(::Type{<: MyPlot}, x, y, z) =  Makie.LScene
+```
+
```

Comment:
```suggestion

or

```julia
Makie.preferred_axis_type(plot::MyPlot) = Makie.LScene # if you need the entire plot object as information
```